src/main/java/ChatReques6.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+/**
+ * @author madhankumar
+ */
+@Setter
+@Getter
+@ToString
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ChatReques6 {
+    private String model;
+    private List<?> messages;
+    private int n;
+    private double temperature;
+
```

Comment:
The method 'newMethod' compares strings using '==' which checks reference equality, not content equality. It should use 'equals' method for string comparison.